client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -73,11 +73,9 @@
             if (card.type === 'search' && card.search && card.search.filter.query) {
                 params = card.search.filter.query;
                 if (card.query) {
-                    if (card.search.filter.query.q) {
-                        params.q = '(' + card.query + ') ' + card.search.filter.query.q;
```

Comment:
@akintolga as I saw you added this code couple months ago, can you please review this changes that I made, are they ok. I didn't understand why on every search, query is added every time o previous query (even when i delete search), maybe it is needed somewhere else, and it shouldn't be deleted.